crates/rpc/src/api.rs

```diff
@@ -7,10 +8,13 @@ use core::net::IpAddr;
 
 use alloy_eips::BlockNumberOrTag;
 use alloy_primitives::{B256, U64};
+#[allow(unused_imports)]
```

Comment:
I _think_ we can be more restrictive here, instead of leaking the import to non-wasm builds

```suggestion
#[cfg(all(target_arch = "wasm32", target_os = "unknown"))]
```

Reply:
I like that, thanks